opentech/apply/review/views.py

```diff
@@ -34,6 +35,10 @@ class ReviewCreateView(CreateView):
 
     def dispatch(self, request, *args, **kwargs):
         self.submission = get_object_or_404(ApplicationSubmission, id=self.kwargs['submission_pk'])
+
+        if not self.submission.phase.has_perm('reivew', request.user):
```

Comment:
`s/reivew/review/g`

Reply:
what happens when the user is a reviewer, but is not assigned to review the submission?